terra-jupyter-base/Dockerfile

```diff
@@ -101,25 +104,20 @@ RUN pip3 -V \
     && pip3 install python-datauri \
     && pip3 install jupyter_contrib_nbextensions \
     && pip3 install jupyter_nbextensions_configurator \
-    && pip3 install markupsafe==2.0.1 \
+    && pip3 install markupsafe==2.1.2 \
     # Avoid broken lower versions: https://github.com/jupyter/nbconvert/pull/1624
     && pip3 install "nbconvert>=6.4.5" \
     # for jupyter_delocalize.py and jupyter_notebook_config.py
     && pip3 install requests \
     && pip3 install firecloud \
     && pip3 install terra-notebook-utils \
     && pip3 install crcmod \
-    # For gcloud alpha storage support.
-    && pip3 install google-crc32c --target /usr/lib/google-cloud-sdk/lib/third_party
-
-# install Cromshell 2.0
-RUN git clone https://github.com/broadinstitute/cromshell.git \
-    && cd cromshell \
-    && git checkout cromshell_2.0 \
-    && pip install .
+    # For gcloud alpha storage support.   
+    && pip3 install google-crc32c --target /usr/lib/google-cloud-sdk/lib/third_party \
+    && pip3 install cromshell==2.0.0
```

Comment:
changes from https://github.com/DataBiosphere/terra-docker/pull/424 for a bug fix

Reply:
Once this PR is merge we should close 424 then :)